core/src/main/java/nl/nn/adapterframework/management/bus/endpoints/TestPipeline.java

```diff
@@ -99,7 +99,7 @@ private BinaryResponseMessage processMessage(IAdapter adapter, String payload, M
 				pls.putAll(threadContext);
 			}
 
-			Date now = new Date();
+			Instant now = Instant.now();
```

Comment:
Hier ook dan maar null null? beetje overbodig en foutgevoelig?

Reply:
Waarom denk je dat dit overbodig/foutgevoelig is?

Reply:
Omdat hij nergens gebruikt wordt behalve bij PipelineSession.updateListenerParameters. Die doet zelf al, wanneer de date null is, hem zetten. Ik denk dan persoonlijk (maar misschien een beetje buiten scope) dat het mooier is die methode te overloaden, en hier updateListenerParameters(pls, messageId, correlationId) van te maken, of hier null null te zetten en dat in een ander pr op te lossen.

Reply:
Ik heb even gekeken en hier is het niet overbodig omdat het de time-sent representeert (correct) welke anders NULL zou blijven.